packages/test/test-service-load/src/runner.ts

```diff
@@ -176,7 +176,7 @@ async function runnerProcess(
 	// Assigning no-op value due to linter.
 	let metricsCleanup: () => void = () => {};
 
-	const optionsOverride = `${driver}${endpoint !== undefined ? `-${endpoint}` : ""}`;
+	const optionsOverride = endpoint ?? driver;
```

Comment:
Make sure that the optionsOverride for the first container matches this. The code for the first container lives here - https://github.com/microsoft/FluidFramework/blob/a4bb4900972621536621f06f27bc7b1f63e5b539/packages/test/test-service-load/src/utils.ts#L166